src/components/Popup/index.stories.tsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable */
+
+import { Meta, StoryFn } from '@storybook/react';
+import Popup, { PopupProps } from '.';
+
+export default {
+  title: 'Components/Popup',
+  component: Popup,
+} as Meta;
+
+const Template: StoryFn<PopupProps> = (args) => <Popup {...args} />;
```

Comment:
```suggestion
const meta: Meta<typeof Popup> = {
  component: Popup,
};
export default meta;

type Story = StoryObj<typeof Popup>;

export const Default: Story = {
   args:{
     message: ...
     }
```
혹시 이런식으로 해보셨을까요?? 저번 프로젝트에서는 이런식으로 했는데 안되나 궁금해서요~